HelpDeskCommonMethods/src/main/java/webdriverbase/AppPage.java

```diff
@@ -91,6 +95,7 @@ public JavascriptExecutor getJavaScriptExecutor() {
 	
 	public void scrolltoElement(String locator) {
 		try {
+			
```

Comment:
An Explicit wait could be defined in line 100 to wait for the element
new WebDriverWait(driver, timeout).until(ExpectedConditions.visibilityOf(ele));